admin/js/common.js

```diff
@@ -562,13 +565,13 @@ var qTranslateX=function(pg)
 								}
 							}
 						}else{
-							continue;
+							continue;
```

Comment:
I think they're Windows line breaks mixed in between the Linux line breaks of the rest of the code. I was unable to remove them from the commit even though I tried with 2 clients (GUI and command line). They were always changed back at commit time.